include/sfx.h

```diff
@@ -235,4 +236,25 @@ typedef enum { MONO_SOUND, STEREO_SOUND } soundMode;
 #define NA_SE_VO_DR_HURT_3 0x85C
 #define NA_SE_VO_DR_HURT_4 0x85D
 #define NA_SE_PL_TELEPORT 0x8BA
-#define NA_SE_CS_BURNING_PHOTOGRAPH 0x8BE
\ No newline at end of file
+#define NA_SE_CS_BURNING_PHOTOGRAPH 0x8BE
+
+// STAGE DRE
+#define NA_SE_SU_SHOOT_PINKBALLS 0x62C
+#define NA_SE_SU_LANDING 0x646
+#define NA_VO_SU_CRYSTAL_2 0x6AF
+#define NA_SE_SU_FLAPING_WINGS 0x6C6
```

Comment:
"flapping". Also I think we have definitive evidence now that sfx IDs are reused https://github.com/Xeeynamo/sotn-decomp/blob/c620451743090a5d6be4c0a22399ffefd87a7027/include/sfx.h#L203

I'm thinking this should be in dre.h since it's specific to DRE.

Reply:
good catch on the typo, i still think we should put all SFXs in sfx.h separating them by overlays, even though some sfx are reused, you'll find that different stages use the same offsets for different sound effects.

Reply:
It's still a good idea to find all SFX on the same file

Reply:
Should the entity ID enums be moved to one shared file? What's the difference there?

Reply:
Discussion continues in discord.